app/lib/actions/katello/capsule_content/remove_orphans.rb

```diff
@@ -9,8 +9,11 @@ def plan(proxy)
           sequence do
             plan_action(Actions::Katello::CapsuleContent::RemoveUnneededRepos, proxy) unless proxy.pulp_master?
 
-            if proxy.pulp3_enabled? && proxy == SmartProxy.pulp_master
-              plan_action(Actions::Pulp3::Orchestration::Repository::RemoveOrphans, proxy)
+            if proxy.pulp3_enabled?
+              if proxy == SmartProxy.pulp_master
+                plan_action(Actions::Katello::Pulp3::Orchestration::Repository::RemoveOrphans, proxy)
+              end
+              plan_action(Actions::Pulp3::CapsuleContent::RemoveUnneededRepos, proxy)
```

Comment:
since there's going to be multiple actions for both the main server and the mirror, i wonder if it makes sense to move all the pulp3 stuff into Actions::Katello::Pulp3::Orchestration::Repository::RemoveOrphans  ?